torchvision/ops/deform_conv.py

```diff
@@ -0,0 +1,70 @@
+from typing import Tuple
+
+import torch
+from torch import nn, Tensor
+from torch.nn.modules.utils import _pair
+from torch.jit.annotations import List
+
+
+def deform_conv(input, offset, weight, stride=(1, 1), pad=(0, 0), dilation=(1, 1), n_parallel_imgs=64):
```

Comment:
Can you name it `deform_conv2d`, because it's a conv2d operator?

Reply:
Also, can you rename `pad` to `padding` to make it consistent with pytorch `conv2d`, and change the order to be `input, weight, offset, ...`?